Changelog.md

```diff
@@ -11,6 +11,7 @@
 - Switched to consistent Font Awesome chevrons for expander icons (#7713)
 - Install Ruby-LSP to allow development inside different IDEs such as VSCode (#7718)
 - Ensure only instructors and admins can link course, as LMS launch MarkUs button made available for all users (#7714)
+- Add API endpoint `add_test_results` that allows independent user submissions of test executions to MarkUs (#7730)
```

Comment:
In terms of naming, we should call this route `add_test_run` to align with the parent object (`TestRun`) that's created